src/runtime_src/core/edge/fragments/xlnk_dts_fragment_versal_mpsoc.dts

```diff
@@ -0,0 +1,27 @@
+
+/{
+        reserved-memory {
```

Comment:
if this device tree file is for used by edge only, we don't need to reserve this memory, right? can you please check the memtopology xclbin built by v++?

Reply:
removed reserved-memory